app/services/platforms/facebook.ts

```diff
@@ -44,11 +44,14 @@ export interface IFacebookLiveVideo {
   id: string;
   stream_url: string;
   title: string;
+  /** @deprecated: doesn't exist on their API docs  */
   game: string;
   description: string;
   permalink_url: string;
   video: { id: string };
   broadcast_start_time: string;
+  planned_start_time?: string;
+  /** custom fields we've created */
```

Comment:
non-blocking comment but `event_params` isn't actually a custom field, it's what is required to send to the new events api: https://developers.facebook.com/docs/live-video-api/guides/scheduling/